package.json

```diff
@@ -283,16 +283,20 @@
       "@fluentui/web-components/@storybook/html",
       "@fluentui/web-components/ts-loader",
       "@fluentui/web-components/ts-loader/**",
-      "@fluentui/web-components/webpack"
+      "@fluentui/web-components/ts-node",
```

Comment:
this looks like bad rebase, ts-node has been unified across the repo https://github.com/microsoft/fluentui/pull/24255/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519